paasta_tools/mesos/master.py

```diff
@@ -166,12 +166,17 @@ def slaves(self, fltr=""):
             itertools.ifilter(
                 lambda x: fltr == x['id'], self.state['slaves'])))
 
-    def _task_list(self, active_only=False):
+    def _task_list(self, active_only=False, include_orphans=False):
```

Comment:
Good question. Would be easy enough to move it "up a level". I did at least make the default to not include them.

I think there's an argument for keeping it here too. Along the lines of: "I'm looking to get all the tasks and don't know that orphans are a thing, I lookup the tasks function here and see the kwarg so I can now think about it and make a decision."